llvm/test/CodeGen/RISCV/and-negpow2-cmp.ll

```diff
@@ -64,3 +64,37 @@ define i1 @test4(i64 %x) {
   %b = icmp eq i64 %a, 0
   ret i1 %b
 }
+
+define i1 @test5(i64 %x) {
```

Comment:
Can you precommit the tests so we can see prior codegen?

Reply:
> Can you precommit the tests so we can see prior codegen?

They are in separate commits in the PR.